### ISSUE TITLE
Docker unable to locate package

### ISSUE THREAD

**Issue by Gabriel0402** [Submitty/Submitty#3657]
**Describe the bug**
When running the install_system.sh script with the build docker file line, there are errors indicating unable to locate packages. 
Errors output as:
Step 3/10 : RUN apt-get install -qqy python python-dev python3 python3-dev libpython3.5
 ---> Running in e2a60695faf1
E: Unable to locate package python
E: Unable to locate package python-dev
E: Unable to locate package python3
E: Unable to locate package python3-dev

Try to modify the Dockerfile with apt-get update && apt-get install -qqy python. Still did not work.

**Expected behavior**
Since the cache is got from apt-get update, then the python packages should be installed correctly.

**To Reproduce**
1. Run install_system.sh
2. See error

**Configuration**
 - OS: Ubuntu 18.04 LTS

**Comment by Gabriel0402** [comment]
Also tried to run docker build with --no-cache, got the following warnings and errors:
W: Problem unlinking the file /var/cache/apt/archives/partial/.apt-acquire-privs-test.SGh534 - IsAccessibleBySandboxUser (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/archives/partial/.apt-acquire-privs-test.CYnak4 - IsAccessibleBySandboxUser (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/archives/partial/.apt-acquire-privs-test.0kbgA3 - IsAccessibleBySandboxUser (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/archives/partial/.apt-acquire-privs-test.WjEmQ2 - IsAccessibleBySandboxUser (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/archives/partial/.apt-acquire-privs-test.4BNt61 - IsAccessibleBySandboxUser (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/archives/partial/.apt-acquire-privs-test.wOFBm1 - IsAccessibleBySandboxUser (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/archives/partial/.apt-acquire-privs-test.EriKC0 - IsAccessibleBySandboxUser (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/archives/partial/.apt-acquire-privs-test.37zTSZ - IsAccessibleBySandboxUser (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/archives/partial/.apt-acquire-privs-test.OFz38Y - IsAccessibleBySandboxUser (13: Permission denied)
....

debconf: delaying package configuration, since apt-utils is not installed
...

Errors were encountered while processing:
 /tmp/apt-dpkg-install-l4ERj5/254-libc6-dbg_2.27-3ubuntu1_amd64.deb
 /tmp/apt-dpkg-install-l4ERj5/255-valgrind_1%3a3.13.0-2ubuntu2.1_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)

**Comment by MasterOdin** [comment]
Are you using a proxy for Vagrant?

**Comment by Gabriel0402** [comment]
I did not use vagrant. Instead, I git clone to local and run the install_system.sh directly.

**Comment by bmcutler** [comment]
@pbailie Can you attempt to reproduce this issue with a new (temporary) VM?

**Comment by pbailie** [comment]
I'll look into this.

**Comment by pbailie** [comment]
I built Submitty twice and did not recreate the error noted in this issue, but I did encounter a different problem.

When I built Submitty in a VM by cloning the repo myself and running `install_system.sh`,  it installed OK and I was able to get to the login page in Chrome.

I deleted my VM, rebuilt it, and used the bootstrap script.  The install crashed as soon as it tried to run configuration.

SSH terminal screenshot:
![Screen Shot 2019-05-07 at 6 52 29 PM](https://user-images.githubusercontent.com/12703624/57338017-cec9a580-70f9-11e9-8a3a-8e3b4ac7edff.png)
